presto-docs/src/main/sphinx/release/release-0.153.rst

```diff
@@ -6,32 +6,154 @@ General Changes
-* Make resource group configuration more flexible. See "SPI Changes" below, and the
-  :doc:`resource groups documentation </admin/resource-groups>`.
+* Fix silent numeric overflow when casting ``INTEGER`` to large ``DECIMAL`` types.
+* Fix issue where ``GROUP BY ()`` would produce no results if the input had no rows.
+* Fix null handling in :func:`array_distinct` when applied to the ``array(bigint)`` type.
+* Fix handling of ``-2^63`` as the element index for :func:`json_array_get`.
+* Fix correctness issue when the input to ``TRY_CAST`` evaluates to null.
+  For types such as booleans, numbers, dates, timestamps, etc., rather than
+  returning null, a default value specific to the type such as
+  ``false``, ``0`` or ``1970-01-01`` was returned.
+* Fix potential thread deadlock in coordinator.
+* Fix rare correctness issue with an aggregation on a single threaded right join when
+  ``task.concurrency`` is ``1``.
+* Fix query failure when casting a map with null elements in values.
+* Fix failure when view column names contain upper-case letters.
+* Fix potential performance regression due to skew issue when
+  grouping or joining on columns of the following types: ``TINYINT``,
+  ``SMALLINT``, ``INTEGER``, ``BIGINT``, ``REAL``, ``DOUBLE``,
+  ``COLOR``, ``DATE``, ``INTERVAL``, ``TIME``, ``TIMESTAMP``.
+* Fix potential memory leak for delete queries.
+* Fix query stats to not include queued time in planning time.
+* Fix spurious log messages when queries are torn down.
+* Remove broken ``%w`` specifier for :func:`date_format` and :func:`date_parse`.
+* Make resource group configuration more flexible.
+  See **SPI Changes** below and :doc:`/admin/resource-groups`.
 * Improve performance of :ref:`array_type` when underlying data is dictionary encoded.
 * Improve performance of ORC reader when decoding dictionary encoded :ref:`map_type`.
+* Improve performance of outer joins with non-equality criteria.
+* Require task concurrency and task writer count to be a power of two.
+* Use nulls-last ordering for :func:`array_sort`.
+* Validate that ``TRY`` is used with exactly one argument.
+* Allow running Presto with early-access Java versions.
+* Allow subqueries in non-equality outer join criteria.
+* Add support for :doc:`/sql/create-schema`, :doc:`/sql/drop-schema`
+  and :doc:`/sql/alter-schema`.
+* Add initial support for correlated subqueries.
+* Add execution support for prepared statements.
+* Add ``DOUBLE PRECISION`` as an alias for the ``DOUBLE`` type.
+* Add :func:`typeof` for discovering expression types.
+* Add decimal support to :func:`avg`, :func:`ceil`, :func:`floor`, :func:`round`,
+  :func:`truncate`, :func:`abs`, :func:`mod` and :func:`sign`.
+* Add :func:`shuffle` function for arrays.
 * Add :doc:`/connector/accumulo`.
 
-SPI Changes
+Web UI Changes
+--------------
+
+* Fix rendering failures in due to null nested data structures.
+* Fix active worker count on cluster overview page to not include coordinator.
```

Comment:
Changed to "Do not include coordinator in active worker count on cluster overview page."